x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/metadata/table.stories.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCard } from '@elastic/eui';
+import { I18nProvider } from '@kbn/i18n-react';
+import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import { decorateWithGlobalStorybookThemeProviders } from '../../../../../../test_utils/use_global_storybook_theme';
+import { Table, Props } from './table';
+
+const mockServices = {
+  data: {
+    query: {
+      filterManager: { filterManagerService: { addFilters: () => {}, removeFilter: () => {} } },
+    },
+  },
+  notifications: { toasts: { toastsService: { addSuccess: () => {} } } },
+  telemetry: () => {},
+};
+
+export default {
+  title: 'infra/Host Details View/Metadata Table',
```

Comment:
Perhaps `Metadata Tab` would represent better this story?

Reply:
Those components can be used also in other contexts (not only as a tab inside the flyout) to keep it more organized I created another folder for the components so it will look like this: 
<img width="238" alt="Screenshot 2023-05-12 at 20 11 55" src="https://github.com/elastic/kibana/assets/14139027/feebc59b-e75e-4e76-b6fd-7bf25f0d8638">